WcaOnRails/spec/factories/users.rb

```diff
@@ -140,6 +140,12 @@
       end
     end
 
+    trait :wfc_leader do
```

Comment:
This shouldn't go here, as you can use `FactoryBot.create :user, :wfc_member, team_leader: true`.